libs/Microsoft.MixedReality.WebRTC.Native/src/pch.h

```diff
@@ -25,13 +25,17 @@
 #define NOMINMAX
 #endif
 
+#define MRS_NODISCARD [[nodiscard]]
+
 #include <windows.h>
 
 #elif defined(MR_SHARING_ANDROID)
 
 #define WEBRTC_POSIX 1
 #define WEBRTC_ANDROID 1
 
+#define MRS_NODISCARD
```

Comment:
This is because of C++14 I assume? I think we will need to remove it entirely, even on Windows, because libwebrtc doesn't support C++17. So far we were compiling it with C++17 anyway but I'd rather be compliant with the Google repo. This can stay as is for now though.

Reply:
Interesting, I was able to put [[nodiscard]] back without compilation errors. As I was working through the build config issues it was generating compiler errors at one point. Android is being compiled with C++17.

Reply:
Ok good to know. I do like the change though, because I want to roll back native to C++14 to be in line with the official repo, since this will be internal code from now (the C++ library will now build on top of it, so can do whatever it wants, and will be C++17). So let's leave that `MRS_NODISCARD` macro.

Reply:
Argh sorry I saw you reverted already. My initial comment was not clear. You can leave it as is.

Reply:
Oh sorry about that.

Reply:
Thanks